### ISSUE TITLE
Don't require GM status for check/contest

### ISSUE THREAD

**Issue by rozzingit** [Arx-Game/arxcode#348]
This was discussed on channel at the time the new check system came out, and Apos agreed (IIRC) that it was probably best. check/contest is actually plenty useful for non-GMed scenes, either events or just regular RP, and encouraging event runners who aren't GMing to just set themselves as GM for a cal event is a bad idea, as staff has specifically had to discourage that in the past because of people setting themselves as a GM and soliciting GM RS for events they are ICly running but that aren't PRPs they're GMing.

Basically, probably better for check/contest and check/contest/here to just not require a GM status. (And deal with anyone trying to spam it or something by telling the individuals to knock it off.)

I just didn't notice until today that it wasn't adjusted at the time and there wasn't an outstanding issue for it, so I'm recording it here so it doesn't get forgotten.

**Comment by TehomCD** [comment]
Although I'll be changing the mechanics of how checks work, check/contest will still be maintained in some form, so this feature still works for Hacktoberfest
